### PR TITLE
openSUSE in System Requirements

### PR DESCRIPTION
**Pull request by cabelo**

### CODE REVIEW

File: docs/articles_en/get-started/install-openvino/install-openvino-linux.rst [hunk 1]
```diff
@@ -33,6 +33,7 @@ If you want to install OpenVINO™ Runtime on Linux, you have the following opti
 * :doc:`Install OpenVINO using PyPI <install-openvino-pip>`
 * :doc:`Install OpenVINO using APT <install-openvino-apt>`
 * :doc:`Install OpenVINO using YUM <install-openvino-yum>`
+* :doc:`Install OpenVINO using ZYPPER <install-openvino-zypper>`
```
Comment (sgolebiewski-intel) [inline docs/articles_en/get-started/install-openvino/install-openvino-linux.rst:36]: Hasn't it been merged with: https://github.com/openvinotoolkit/openvino/pull/24208/files#diff-e7b442908757c28e88d289cd4a602229771908d9e80263fbbf1360ef98dfcff3 
?

**Review by sgolebiewski-intel — APPROVED** [review]

**Comment by sgolebiewski-intel** [comment]
build_jenkins

**Comment by cabelo** [comment]
> build_jenkins

Thanks

**Comment by cabelo** [comment]
> build_jenkins

@tsavina or @ilya-lavrenov Do I need to do anything else?
Thanks

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**